processor-api/src/main/java/com/synopsys/integration/alert/processor/api/filter/JobNotificationMapper.java

```diff
@@ -46,17 +48,12 @@ public JobNotificationMapper(ProcessingJobAccessor processingJobAccessor) {
      * Filter by Policy name (From notification if applicable)
      * @param detailedContents List of notifications that will be iterated over and applied to jobs that are found
      * @param frequencies      an Additional filter to specify when querying data from the DB
-     * @return a {@code StatefulAlertPagedModel} where a page of distributions job is used to map to a list of notifications that were passed in.
+     * @return a {@code StatefulAlertPage} where a page of distributions job is used to map to a list of notifications that were passed in.
      */
-    public StatefulAlertPagedModel<FilteredJobNotificationWrapper> mapJobsToNotifications(List<DetailedNotificationContent> detailedContents, List<FrequencyType> frequencies) {
+    public StatefulAlertPage<FilteredJobNotificationWrapper> mapJobsToNotifications(List<DetailedNotificationContent> detailedContents, List<FrequencyType> frequencies) {
         AlertPagedModel<FilteredJobNotificationWrapper> pageOfJobsToNotification = mapPageOfJobsToNotification(detailedContents, frequencies, INITIAL_PAGE_NUMBER, PAGE_SIZE);
```

Comment:
Doesn't this also need to change if we are trying to avoid using the AlertPagedModel for FileredJobNoificationWrapper?